### PR TITLE
Add draw calls to non-indexed buffers

### PR DESCRIPTION
**Pull request by benaadams**

### CODE REVIEW

File: src/renderers/WebGLRenderer.js [hunk 1]
```diff
@@ -1135,56 +1135,93 @@ THREE.WebGLRenderer = function ( parameters ) {
...
 
+					if ( geometry instanceof THREE.InstancedBufferGeometry ) {
+
+						THREE.error( 'THREE.WebGLRenderer.renderMesh: cannot use drawCalls with THREE.InstancedBufferGeometry.' );
+						return;
+
+					} else {
+
+						_gl.drawArrays( mode, offsets[i].index, offsets[i].count );
+
```
Comment (WestLangley) [inline src/renderers/WebGLRenderer.js]: _gl.drawArrays( mode, offsets[ i ].start, offsets[ i ].count );  ?

**Comment by WestLangley** [comment]
I think this is going to have to be checked. There seems to be a number of problems... But it is late for me, so it will have to wait... : - )

**Comment by benaadams** [comment]
I did qualify it will hopefully ;-)

Don't have a test case to try it with

**Comment by WestLangley** [comment]
What to do about `drawcalls.index` in this PR? I assume it does not apply?

Again, it is a poorly-named property, IMO. `drawcalls.indexOffset` would be better.

**Comment by benaadams** [comment]
> What to do about drawcalls.index in this PR? I assume it does not apply?

Yeah, doesn't really mean anything in this context.

**Comment by mrdoob** [comment]
Sweet! Thanks!

**Comment by WestLangley** [comment]
As I said, 

> Again, it is a poorly-named property, IMO. `drawcalls.indexOffset` would be better.

**Comment by benaadams** [comment]
> Again, it is a poorly-named property, IMO. drawcalls.indexOffset would be better.

@WestLangley Lot of changes to do that change #6498

**Comment by WestLangley** [comment]
@benaadams Thanks!

**Pull request merged**